clients/rust/src/generated/instructions/deactivate_stake.rs

```diff
@@ -144,7 +145,8 @@ impl DeactivateStakeBuilder {
 
 /// `deactivate_stake` CPI accounts.
 pub struct DeactivateStakeCpiAccounts<'a, 'b> {
-    /// Validator stake account (pda of `['stake::state::stake', validator, config]`)
+    /// Validator stake account (pda of `['stake::state::stake', validator,
+    /// config]`)
```

Comment:
Ahh, some of these are a bit rough after formatting, but this can be handled another time!